### DISCUSSION TITLE
Mystery Dust silent command argument

### DISCUSSION THREAD

**Discussion by Thornbush42** [yapzhenyie/GadgetsMenu#868] [Ideas]
Hi! Loving the premium plugin :D
On our server we have a setup to give Mystery Dust to a player every 10m/1h/1d - but when we give them it they receive a message (or blank line if removed from the messages.yml as a semi-solution) which we don't require or want.
I suggest a possible "-s" argument in (or at the end of) the command to not send a message to the receiving player, much like other plugins have.
Thanks for reading all the way through :D

**Comment by yapzhenyie** [comment]
![1](https://user-images.githubusercontent.com/13622737/128619006-dd8836cc-2f78-42e3-9fa3-2e158e37e117.PNG)

**Reply by Thornbush42** [reply]
Ah thanks! I was unaware this was already a thing as it does not list it in your wiki. Many thanks anyhow!
